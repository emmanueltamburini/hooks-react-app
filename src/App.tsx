function App() {
  return (
    <>
      <h1>BoilerPlate</h1>
    </>
  )
}

export default App
