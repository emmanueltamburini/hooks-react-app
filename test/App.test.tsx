import React from 'react';
import App from '../src/App'
import { render, screen } from '@testing-library/react';

describe('Test in GifApp', () => {
    test('should be equal to snapshot', () => {
        const {container} = render(<App />);
        expect(container).toMatchSnapshot();
    });

    test('should have boilerplate word', () => {
        render(<App />);
        expect(screen.getByText('BoilerPlate'));
    });
})