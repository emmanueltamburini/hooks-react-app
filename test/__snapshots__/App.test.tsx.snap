// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test in GifApp should be equal to snapshot 1`] = `
<div>
  <h1>
    BoilerPlate
  </h1>
</div>
`;
